doc/compiling.md

```diff
@@ -215,7 +215,8 @@ To do this with only one simple command, type:
 1. Change into the project source directory: `cd stlink`
 2. Run `make clean` to clean remnants of any previous builds.
 3. Run `make release` to create the _Release_ target
-4. Run `make debug` to create the _Debug_ target (_optional_)<br />
+4. Run `make install` to full install the package with complete system integration. This might require sudo permissions.
```

Comment:
In case of Linux, superuser permissions may be needed as well. Consider adding this note there.

Perhaps the build instructions for the these operating systems could be merged?

Reply:
Common requirements etc. are different for Linux and macOS, so I think merging the installation instructions section make things more difficult to follow/find. Did add the node for linux though!

Reply:
I would not merge both either. The separation was also intended originally to guide users to the specific instruction they need for their type of system, in relation to the required setup, as @jdbruijn realised already.